lambdas/service/app.py

```diff
@@ -810,69 +885,79 @@ def start_manifest_generation():
                     status_code=301 if wait_time else 302)
 
 
-@app.route('/fetch/manifest/files', methods=['GET'], cors=True)
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, spec={
+    'description': 'Initiate and check status of a manifest generation job, returning a 200 response with simulated '
+                   'headers in the body.<p>'
+                   'If the client receives a response body with the `Status` field set to 301, the client should wait '
+                   'the number of seconds specified in `Retry-After` and then request the URL given in the `Location` '
+                   'field. The URL will point back at this endpoint so the client should expect a response of the same '
+                   'shape. Note that the actual HTTP response is of status 200, only the `Status` field of the body '
+                   'will be 301. The intent is to emulate HTTP while bypassing the default client behavior which, in '
+                   'most web browsers, is to ignore `Retry-After`. The response described here is intended to be '
+                   'processed by client-side Javascript such that the recommended delay in `Retry-After` can be '
+                   'handled in Javascript rather that relying on the native implementation by the web browser.<p>'
+                   'The client should request the URL given in the `Location` field. The URL will point to a different '
+                   'service and the client should expect a response containing the actual manifest. Currently the '
+                   '`Location` field of the final response is a signed URL to an object in S3 but clients should not '
+                   'depend on that.',
+    'parameters': [
+        {
+            'in': 'query',
+            'name': 'filters',
+            'description': 'Filters to be applied when generating the manifest',
+            'required': False,
+            'schema': {
+                'type': 'string'
+            }
+        },
+        {
+            'in': 'query',
+            'name': 'format',
+            'description': 'The desired format of the output. Possible values are `compact` (the default) for a '
```

Comment:
Instead of duplicating the out of date description here, we can just use python's reuse mechanisms and make a variable or something.